### PR TITLE
📖 Add link to videos in the Getting started section

### PR DESCRIPTION
**Pull request by oscr**
I would like to suggest adding a link to the videos explaining CAPI to the Getting started section. I personally found them very helpful and I think they should be made more visible to help others getting started with the project to find them.

### CODE REVIEW

File: docs/book/src/developer/providers/webhooks.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Defaulting webhooks are an implementation of a [Kubernetes mutating webhook](htt
 ## Conversion webhooks
 Conversion webhooks are what allow Cluster API to work with multiple API types without requiring different versions. It does this by converting the incoming version to a `Hub` version which is used internally by the controllers. To read more about conversion see the [Kubebuilder documentation](https://book.kubebuilder.io/multiversion-tutorial/conversion.html)
 
-For a walkthrough on implementing conversion webhooks see the video in the [Developer Guide](https://cluster-api.sigs.k8s.io/developer/guide.html?highlight=video#videos-explaining-capi-architecture-and-code-walkthroughs)
+For a walkthrough on implementing conversion webhooks see the video in the [Developer Guide](../guide.md?highlight=video#videos-explaining-capi-architecture-and-code-walkthroughs)
```
Comment (killianmuldoon) [inline docs/book/src/developer/providers/webhooks.md:14]: I'm not certain why but I think:
```
highlight=video
```

Doesn't work with MD (could just be my local setup?)

For safety let's do what we have elsewhere:
```suggestion
For a walkthrough on implementing conversion webhooks see the video in the [Developer Guide](../guide.md#videos-explaining-capi-architecture-and-code-walkthroughs)
```

**Review by killianmuldoon — COMMENTED** [review]
Let's backport this to the 1.2 version of the book too 🙂

File: docs/book/src/developer/providers/webhooks.md (hunk 1 above)
Comment (oscr) [reply docs/book/src/developer/providers/webhooks.md]: Sounds great. Fixed

**Review by killianmuldoon — APPROVED** [review]
/lgtm

**Pull request merged**